exonum-java-binding/cryptocurrency-demo/start-cryptocurrency-demo.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# prints a section header
+function header() {
+    local title=$1
+    local rest="========================================================================"
+    echo
+    echo "===[ ${title} ]${rest:${#title}}"
+    echo
+}
+
+set -eu -o pipefail
+
+header "DETECTING ENVIRONMENT"
+
+# Check whether the Exonum Java App is installed.
+EXONUM_JAVA_APP="exonum-java"
+command -v ${EXONUM_JAVA_APP} >/dev/null 2>&1 || { echo >&2 "Please install the Exonum Java App and make sure that 'exonum-java' binary is available via PATH. Aborting."; exit 1; }
+
+# Check that path to the JVM library is set. We cannot just set the LD_LIBRARY_PATH outside the script because of OSX SIP limitation.
+if [ -z "${LIB_JVM_PATH+x}" ];
+then
+    echo "ERROR: LIB_JVM_PATH variable is not set. Please make sure it points to the directory where the JVM dynamic library (libjvm.[so|dylib]) resides."
+    exit 1
+fi
+
+export LD_LIBRARY_PATH=${LIB_JVM_PATH}
+echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}"
+
+# Find the latest version of artifact and build if not found.
+ARTIFACT_PATH="$(find ./target -type f -name exonum-java-binding-cryptocurrency-demo-*-artifact.jar)"
+
+if [ -z "${ARTIFACT_PATH}" ];
+then
+    mvn package
+    ARTIFACT_PATH="$(find ./target -type f -name exonum-java-binding-cryptocurrency-demo-*-artifact.jar)"
+fi
+echo "ARTIFACT_PATH=${ARTIFACT_PATH}"
+
+# Prepare the services configuration file.
+SERVICES_CONFIG_FILE="services.toml"
```

Comment:
is it a correct name? The doc says it should be  `ejb_app_services.toml`

Reply:
AFAIK the name of the file is correct after renaming `ejb-app` into `exonum-java` and most probably we have outdated docs.

Reply:
I see https://github.com/exonum/exonum-doc/pull/312